src/622/modules/password/index.test.js

```diff
@@ -0,0 +1,7 @@
+const password = require('./index')
```

Comment:
Este aqui, pelo jeito tbm não deveria estar aqui.. eheheheh

Reply:
Corrigido no commit [fix: removing the password module from this branch](https://github.com/dojo-coders/keep-talking-defuser-helper/pull/22/commits/c89b3ce8046845280a2d52320106afb32c832d8e)